frontend/index.html

```diff
@@ -101,6 +101,11 @@ <h2 id='game-over-header' class="hidden">Game Over!</h2>
               </div>
             </div>
 
+            <div class="row justify-content-center">
+              <div id="typo-alert" class="hidden">
```

Comment:
maybe this could be changed to an alert modal thingy in the future?

Reply:
ooh have you been using that for the leaderboard? New to me, would love to learn